lib/ProductOpener/Packaging.pm

```diff
@@ -505,6 +505,15 @@ sub apply_rules_to_augment_packaging_component_data ($product_ref, $packaging_re
 		$packaging_ref->{"shape"} = "en:coffee-capsule";
 	}
 
+	# If the shape is bottle and the material is glass, mark recycling as recycle
+	if (    (defined $packaging_ref->{"shape"})
+		and ($packaging_ref->{"shape"} eq "en:bottle")
+		and (defined $packaging_ref->{"material"})
+		and ($packaging_ref->{"material"} eq "en:glass"))
+	{
+		$packaging_ref->{"recycling"} = "en:recycle";
+	}
```

Comment:
Only apply the rule if there was no value provided.

Also as the test showed there is a [en:recicle-in-glass-bin](https://github.com/openfoodfacts/openfoodfacts-server/blob/main/taxonomies/packaging_recycling.txt#L41) which seems even more appropriate.


```suggestion
	if (    (! defined $packaging_ref->{"recycling"})
	    and (defined $packaging_ref->{"shape"})
		and ($packaging_ref->{"shape"} eq "en:bottle")
		and (defined $packaging_ref->{"material"})
		and ($packaging_ref->{"material"} eq "en:glass"))
	{
		$packaging_ref->{"recycling"} = "en:recycle-in-glass-bin";
	}
```

Reply:
I would prefer that we stick with "en:recycle". I'm not sure if we will keep the "recycle in XYZ bin" in the long run.